satpy/readers/seviri_base.py

```diff
@@ -980,3 +981,23 @@ def mask_bad_quality(data, line_validity, line_geometric_quality, line_radiometr
     line_mask = line_mask[:, np.newaxis]
     data = data.where(~line_mask, np.nan).astype(np.float32)
     return data
+
+
+def round_nom_time(dt=None, date_delta=None):
+    """Round a datetime object to a multiple of a timedelta.
+
+    dt : datetime.datetime object, default now.
+    dateDelta : timedelta object, we round to a multiple of this, default 1 minute.
```

Comment:
3. - > This is because the commit check up prevented me to call for a function in the call of a function( here datetime.now()) So i used teh None type and then check if value is None get the defaulf values as now and timedelta. 
```python 
if dt is None:
        dt = datetime.now()
    seconds = (dt - dt.min).seconds
    if date_delta is None:
        date_delta = timedelta(minutes=1)
```

Reply:
4.-> The idea was to be able to test the function without having to add anything. The default behavior should give you "now" rounded to the minutes. 
```python

round_nom_time()
Out[149]: datetime.datetime(2023, 4, 3, 8, 14)
```

Reply:
1. and 2. resolved by https://github.com/pytroll/satpy/pull/2420/commits/d64980735a7669398e2f626c715a5c54ae024937